src/main/java/kernitus/plugin/OldCombatMechanics/module/ModuleOldArrows.java

```diff
@@ -0,0 +1,67 @@
+package kernitus.plugin.OldCombatMechanics.module;
+
+import kernitus.plugin.OldCombatMechanics.OCMMain;
+import org.bukkit.entity.Arrow;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.event.entity.ProjectileLaunchEvent;
+import org.bukkit.projectiles.ProjectileSource;
+import org.bukkit.util.Vector;
+
+/**
+ * Prevents players using new arrow mechanics
+ */
+public class ModuleOldArrows extends Module {
+
+
+    public ModuleOldArrows(OCMMain plugin){
+        super(plugin, "old-arrows");
+    }
+
+    @EventHandler(priority = EventPriority.HIGHEST)
+    public void onProjectileHit(EntityDamageByEntityEvent e){
+        if(!(e.getEntity() instanceof Player)){
+            return;
+        }
+
+        Player player = (Player) e.getEntity();
+
+        if(!isEnabled(player.getWorld())){
+            return;
+        }
+
+        if(!(e.getDamager() instanceof Arrow)){
+            return;
+        }
+
+        Arrow arrow = (Arrow) e.getDamager();
+
+        if (arrow.hasCustomEffects() && module().getBoolean("clear-effects")){
+            arrow.clearCustomEffects();
+        }
+        e.setDamage(arrow.getDamage() * module().getDouble("damage-multiplier"));
+    }
+
+    @EventHandler(priority = EventPriority.HIGHEST)
+    public void onProjectileLaunch(ProjectileLaunchEvent e){
```

Comment:
This is a redundant feature and should be removed. See: https://github.com/kernitus/BukkitOldCombatMechanics/blob/master/src/main/java/kernitus/plugin/OldCombatMechanics/module/ModuleDisableProjectileRandomness.java

If the feature of velocity multipliers is needed, that should probably be added to the other module.